Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -27,6 +27,21 @@ def build_error_checks
         ]).freeze
       end
 
+      def check_for_non_prefixed_findutils
```

Comment:
Not sure why this is not a problem on Linux. Related commit: af62ec0842cb7eb54d2a9bfddcf0ca2b54f46bc3
was pushed directly to Linuxbrew.

```
$ git --no-pager show -s --format=fuller af62ec0842cb7eb54d2a9bfddcf0ca2b54f46bc3
commit af62ec0842cb7eb54d2a9bfddcf0ca2b54f46bc3
Author:     Shaun Jackman <sjackman@gmail.com>
AuthorDate: Tue Mar 1 12:36:37 2016 -0800
Commit:     Shaun Jackman <sjackman@gmail.com>
CommitDate: Tue Mar 1 12:36:37 2016 -0800

    doctor: Fix findutils for Linuxbrew
    
    Disable check_for_non_prefixed_findutils for Linuxbrew.
    Warning: Putting non-prefixed findutils in your path can cause
    python builds to fail.
```

Reply:
`findutils` are always installed `--with-default-names` enabled on Linuxbrew.